cmake/Modules/Findblis.cmake

```diff
@@ -0,0 +1,42 @@
+SET(BLIS_INCLUDE_SEARCH_PATHS
+	~/amd-blis-mt/include/blis
+)
+
+SET(BLIS_LIB_SEARCH_PATHS
+	~/amd-blis-mt/lib
```

Comment:
same here

Reply:
These changes are addressed in our recent 2 commits. Kindly refer to latest files. We have removed the AMD specific names as per your suggestion.